themes/schema.json

```diff
@@ -185,10 +185,28 @@
               "description": "https://ohmyposh.dev/docs/configure#postfix",
               "default": " "
             },
+            "include_folders": {
+              "type": "array",
+              "title": "If specified, segment will only render in these folders",
+              "description": "https://ohmyposh.dev/docs/configure#include--exclude-folders",
+              "default": [],
+              "items": {
+                "type": "string"
+              }
+            },
+            "exclude_folders": {
+              "type": "array",
+              "title": "Exclude rendering in these folders",
+              "description": "https://ohmyposh.dev/docs/configure#include--exclude-folders",
+              "default": [],
+              "items": {
+                "type": "string"
+              }
+            },
             "ignore_folders": {
               "type": "array",
-              "title": "Ignore rendering in these folders",
-              "description": "https://ohmyposh.dev/docs/configure#ignore-folders",
+              "title": "Obsolete - please use `exclude_folders` instead",
```

Comment:
You can use the `deprecated ` keyword for this

Reply:
I actually tried that, but it had no noticeable effect in my editor (VS code).  I also tried upgrading the schema to `"$schema": "https://json-schema.org/draft/2020-12/schema"`.  I'll give it another shot.  Have we used the `deprecated` keyword previously in OMP?

Reply:
no, not yet. I was looking for this for something else but never saw its effect in vscode (or anywhere else). We can add it and keep the title for context.

Reply:
This is done